algosdk/atomic_transaction_composer.py

```diff
@@ -2,7 +2,17 @@
 import copy
 from abc import ABC, abstractmethod
 from enum import IntEnum
-from typing import Any, List, Optional, Tuple, TypeVar, Union
+from typing import (
```

Comment:
I'd appreciate a closer review of this file because I am not yet very familiar with the ABI types and I found it difficult to determine the types of certain variables from their context.